app/assets/stylesheets/app/profile.scss

```diff
@@ -0,0 +1,150 @@
+$team-link-color: #5e9aff;
+$dark-text-color: #49465e;
+$darker-text-color: #5b596e;
+$border-color: #e7e4ed;
+
+.profile {
+  &__section {
+    &+.profile__section {
+      margin-top: 5%;
+    }
+  }
+
+  &__header {
+    flex: 1;
+    padding: 0 2%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+  }
+
+  &__picture {
+    flex: 2;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 2% 0;
+
+    img {
+      flex: 1;
+      object-fit: fill;
+      max-width: 25%;
+      border-radius: 50%;
+      overflow: hidden;
+    }
+  }
+
+  &__info {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__name {
+    flex: 1;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: $dark-text-color;
+    font-size: 200%;
+  }
+
+  &__social-links {
+    flex: 1;
+    display: flex;
+    align-items: center;
+
+    a {
+      text-decoration: none;
+      color: inherit;
+    }
+
+    a:last-of-type {
+      margin-left: 1em;
+    }
+  }
+
+  &__link-icon {
+    &--padded {
+      margin-right: .5em;
+    }
+  }
+
+  &__link {
+    display: flex;
+    flex: 1;
+    font-size: 115%;
+  }
+
+  &__content {
+    display: flex;
+    flex: 2;
+    flex-direction: row;
+    padding: 0 2%;
+
+  }
+
+  &__score {
+    color: $dark-text-color;
+    font-size: 200%;
+  }
+
+  &__text {
+    color: $darker-text-color;
+  }
+
+  &__column {
+    flex: 1;
+    &+.profile__column {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector `profile__column` should be written in lowercase with hyphens<br>Unnecessary parent selector (&)